tests/setup-runtime-dependencies.cmd

```diff
@@ -35,6 +35,12 @@ goto Usage
 if not defined __OutputDir goto Usage
 if not defined __Arch goto Usage 
 
+REM Check if the platform is supported
+if /i %__Arch% == "arm" (
+    echo No runtime dependencies for Arm32.
+    exit /b 0
+    )
```

Comment:
could you also add the same condition for arm64. It fails to restore the package for arm64 currently as it is not available. This does not fail the build though